### PR TITLE
Add test fixture for the wait component 

### PR DESCRIPTION
**Pull request by Blackbaud-DiHuynh**
addressing issue #105 Any feedback is appreciated!

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-indicators?_sp.spa=pr-add-wait-fixture](https://developer.blackbaud.com/skyux-indicators?_sp.spa=pr-add-wait-fixture)

File: src/app/public/testing/wait-fixture.spec.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+} from '@angular/core';
+
+import {
+  TestBed
+} from '@angular/core/testing';
+
+import {
+  SkyWaitModule
+} from '@skyux/indicators';
+import { SkyWaitFixture } from './wait-fixture';
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/testing/wait-fixture.spec.ts:12]: Import styling is off.

**Review by Blackbaud-SteveBrush — COMMENTED** [review]
Looks good! One small change.

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-indicators?_sp.spa=pr-add-wait-fixture](https://developer.blackbaud.com/skyux-indicators?_sp.spa=pr-add-wait-fixture)

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Pull request merged**